WatchDogCore/WatchDog/src/nl/tudelft/watchdog/core/logic/interval/IntervalStatisticsBase.java

```diff
@@ -0,0 +1,248 @@
+package nl.tudelft.watchdog.core.logic.interval;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.joda.time.DateTime;
+import org.joda.time.Duration;
+
+import nl.tudelft.watchdog.core.logic.document.DocumentType;
+import nl.tudelft.watchdog.core.logic.interval.intervaltypes.EditorIntervalBase;
+import nl.tudelft.watchdog.core.logic.interval.intervaltypes.IDEOpenInterval;
+import nl.tudelft.watchdog.core.logic.interval.intervaltypes.IntervalBase;
+import nl.tudelft.watchdog.core.logic.interval.intervaltypes.PerspectiveInterval;
+import nl.tudelft.watchdog.core.logic.interval.intervaltypes.ReadingInterval;
+import nl.tudelft.watchdog.core.logic.interval.intervaltypes.TypingInterval;
+import nl.tudelft.watchdog.core.logic.interval.intervaltypes.UserActiveInterval;
+import nl.tudelft.watchdog.core.logic.interval.intervaltypes.PerspectiveInterval.Perspective;
+
+/** Gathers and calculates statistics on interval length. */
```

Comment:
I don't find this (legacy) comment very accurate. How about "Gathers and calculates statistics on developer behavior in a variable time frame." ?